local_planner/src/nodes/planner_functions.cpp

```diff
@@ -498,15 +530,15 @@ void costFunction(float e_angle, float z_angle, float obstacle_distance,
   // distance cost
   distance_cost = 0.0f;
   if (obstacle_distance > 0.0f) {
-    distance_cost = 12000.0f / obstacle_distance;
+    distance_cost = 700.0f / obstacle_distance;
   }
 
   // combine costs
   other_costs = 0.0f;
-  other_costs =
-      yaw_cost + cost_params.height_change_cost_param_adapted * pitch_cost_up +
-      cost_params.height_change_cost_param * pitch_cost_down + yaw_cost_smooth +
-      pitch_cost_smooth + heading_cost + distance_cost;
+  other_costs = yaw_cost +
```

Comment:
So distance_cost was being added twice before then?

Reply:
yes, this was a bug